src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -68,6 +68,8 @@
  */
 public class DeclarationGenerator {
 
+  public static final String INSTANCE_CLASS_SUFFIX = "_Instance";
```

Comment:
Agreed. I did a quick check against the closure ui libs and looks good.

Removed 'public'.